recipes/law/meta.yaml

```diff
@@ -0,0 +1,92 @@
+{% set name = "law" %}
+{% set version = "0.0.41" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 3b6e457bb99da69ddab03e91a80057958ee9477dba7bb6f8823059b2cbce19b3
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: true  # [win]
+
+requirements:
+  host:
+    - python
+    - pip
+    - six >=1.10
+    - luigi >=2.8.2,<3  # [py<=27]
```

Comment:
Is this actually being used on 2.7?

Reply:
Sadly, yes. Parts of the physics research community is still relying on 2.7 ..